src/daft-micropartition/Cargo.toml

```diff
@@ -18,6 +18,7 @@ pyo3 = {workspace = true, optional = true}
 pyo3-log = {workspace = true}
 serde = {workspace = true}
 snafu = {workspace = true}
+tokio = {workspace = true}
```

Comment:
Where does this new `tokio` dependency for `daft-micropartition` come from? Didn't see it on a cursory glance of the below modules.

Reply:
Ah I see the runtime handle arg now! So it's just used for typing for an arg.

Reply:
Only for type annotating the helper `_read_parquet_into_loaded_micropartition` helper 😬